modules/nw-multus-add-pod.adoc

```diff
@@ -73,6 +65,12 @@ spec:
     image: centos/tools
 ----
 
+. Create the Pod.
```

Comment:
s/Create the Pod./Create the Pod by running the following command:/

Reply:
changed.